### ISSUE TITLE
No single pointer is provided to drag the "columns headers" to the required position in the "Keyboard shortcuts" Table: A11y_Visual Studio Code Client_Customizations and settings_Keyboard Shortcuts_Dragging movements.

### ISSUE THREAD

**Issue by v-jagansai** [microsoft/vscode#224152]
### GIT Hub Tags:
#A11yWCAG2.2; #A11yTCS; #A11ySev2; #Visual Studio Code Client; #BM_Visual Studio Code Client_Win32_JULY2024; #DesktopApp; #WCAG2.5.7; #FTP; #A11yMAS; #Dragging Movements; #Win32;

### Environment and OS details:
Version 24H2(OS build 26120.1252)
Visual Studio code Insider 

### Reproduction Steps:

1. Open Visual Studio Code Insiders editor
2. Tab Navigate to setting icon.
3. Select "Keyboard shortcuts options" and press enter or Press keyboard shortcut as command K, Command S
4. Keyboard shortcut table gets opened.
5. Observe that No single pointer is provided to drag the "columns headers" to the required position in the "Keyboard shortcuts" Table.
6. Actual behavior:
7. No single pointer is provided to drag the "columns headers" to the required position in the "Keyboard shortcuts" Table.

### Expected behavior:
There should be single pointer provided to drag the "columns headers" to the required position in the "Keyboard shortcuts" Table.

### User Impact:
People with motor impairments who cannot operate content because the only way to actuate a function is by dragging a target element from its initial position to some other position will face difficulty in accessing the application.

### Attachments:
[Recording-20240729_120948.webm](https://github.com/user-attachments/assets/56432941-4294-4a2e-9c58-a6137dd43ca3)

**Comment by Samhitha66** [comment]
#Rev:samhithaks;
